os/include/tinyara/wdog.h

```diff
@@ -157,6 +157,7 @@ struct wdog_s {
 	uint8_t flags;				/* See WDOGF_* definitions above */
 	uint8_t argc;				/* The number of parameters to pass */
 	uint32_t parm[CONFIG_MAX_WDOGPARMS];
+	int pid;					/* The pid of process which creates wdog timer */
```

Comment:
When CONFIG_DEBUG is disabled, this is not used. Let's cover this with the conditional.